src/main/java/com/WebController/EventController.java

```diff
@@ -33,9 +33,12 @@
         PatientEventAssociation association;
         boolean verified =false;
         if (principal!=null){ // check if its a verified user
-            verified = this.userRepo.findUserByEmailAddressIgnoreCase(principal.getName()).isVerified();
+            verified = true;
```

Comment:
- I am confused by this. I don't know what Principal does. Why is verified true?
- Why do we need to specify "consumes="application/json""?

- ultimately we still want to get rid of this method, right? just add events one by one, creating a new patient for the first event, and if a sessionid is provided add the event to an existing patient,

Reply:
Principal means a logged in User in spring. If it's not null, which means the person has our account, then we treat information as verified information. 
https://docs.oracle.com/javase/6/docs/api/java/security/Principal.html?is-external=true
I deleted "consumes="application/json". 
Untimately yes, but for now we still don't have user sessions yet. We can delete this method once we have user session in place.